R/style_guides.R

```diff
@@ -206,7 +204,8 @@ create_style_guide <- function(initialize = default_style_guide_attributes,
     # transformer options
     use_raw_indention,
     reindention
-  )
+  ) %>%
+    map(compact)
```

Comment:
Since we changed the default values of `create_style_guide`, now all elements of the list are lists and we can map over them with compact. However, this seems a breaking API change to me. If we keep `initialize = default_style_guide_attributes`, we can't map over them (at least not all of them).